layouts/shortcodes/prismjs.html

```diff
@@ -0,0 +1,27 @@
+{{ $inner := replaceRE "^\n" "" .Inner }}
+{{ if len .Params | eq 0 }}
+  <pre><code>{{ $inner }}</code></pre>
+{{ else }}
+  {{ if .IsNamedParams }}
+    <pre
+      class="
+      {{ if .Get "lang" }}language-{{ .Get "lang" }} {{ end }}
+      {{ if .Get "line-numbers" }}line-numbers {{ end }}
+      {{ if .Get "command-line" }}command-line {{ end }}
+      "
+      {{/* line highlight plugin */}}
+      {{ if .Get "line" }} data-line={{ .Get "line" }} {{ end }}
```

Comment:
```suggestion
      {{ if .Get "line" }} data-line={{ .Get "line" }}{{ end }}
```

Reply:
I'd reduce the space (or... I think this should work `{{- end }}`) for this line and for all below.

Reply:
This tripped me up even when I was writing the shortcode. When I close the space as suggested, Hugo shoots this error:

```
Error: Error building site: "/home/joel/blog/terminal-hack/content/post/1.html:8:1": failed to render shortcode "prismjs": failed to process shortcode: "/home/joel/blog/terminal-hack/layouts/shortcodes/prismjs.html:15:12": execute of template failed: html/template:shortcodes/prismjs.html:15:12: {{if}} branches end in different contexts: {stateAttr delimSpaceOrTagEnd urlPartNone jsCtxRegexp attrNone elementNone <nil>}, {stateTag delimNone urlPartNone jsCtxRegexp attrNone elementNone <nil>}
```

Same error on every line from 13 onwards where I make the same change. Any clue why this trips but line 8 does not?

`Hugo Static Site Generator v0.62.2-83E50184 openbsd/amd64 BuildDate: 2020-01-05T18:53:23Z`

Reply:
Probably it's because you miss parenthesis. I think the code should look like: `{{ if .Get "line" }} data-line="{{ .Get "line" }}"{{ end }}`

Reply:
Yikes. That is super embarrassing 😲

I just pushed a1c203c which cleans up whitespace and adds quotations. Using the shortcode like this:

```
{{< prismjs lang="none" line="7,8" line-numbers="true" command-line="true" start="5" prompt="user@host#" output="2-7" >}}
```

Results in HTML like this:

```
    <pre class="language-none line-numbers command-line" data-line="7,8" data-start="5" data-prompt="user@host#" data-output="2-7"
      ><code class="language-none"
      >this is my highlighted code here, etc, etc
```

Online example [here](http://hugo-terminal.s3-website-us-west-2.amazonaws.com/one/).